pkg/config/rhssoUser.go

```diff
@@ -55,7 +55,7 @@ func (r *RHSSOUser) GetReplicasConfig(inst *integreatlyv1alpha1.RHMI) int {
 		return 1
 	}
 
-	if inst.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManagedApi) {
+	if inst.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManagedApi) || inst.Spec.Type == string(integreatlyv1alpha1.InstallationTypeMultitenantManagedApi) {
```

Comment:
Could you use the IsRHOAM function here?

Reply:
Good point, thanks Laura!

Reply:
Sorry, forgot that doing it gave me errors before. Can't do it due to https://github.com/integr8ly/integreatly-operator/blob/master/pkg/resources/backup.go using pkg/config

Reply:
ah the old import cycle error https://stackoverflow.com/questions/28256923/import-cycle-not-allowed

Reply:
resources/backup imports the config package , so the config package can't import the resources package as it creates a Circular Dependency. moving installTypesHelper.go  to a new package would resolve the issue but not worth it to solve a single line of code

Reply:
It probably makes more sense to put these functions next to where the installTypes are defined? https://github.com/integr8ly/integreatly-operator/blob/c500b9c3ea76b23be11b3fb33005f6c587875221/apis/v1alpha1/rhmi_types.go#L44-L48

which may also address this import loop?